src/libxively/gateway/xi_gw_glue_layer.c

```diff
@@ -0,0 +1,123 @@
+/* Copyright (c) 2003-2018, LogMeIn, Inc. All rights reserved.
+ *
+ * This is part of the Xively C Client library,
+ * it is licensed under the BSD 3-Clause license.
+ */
+
+#include <xi_gw_glue_layer.h>
+#include <xi_layer_api.h>
+#include <xively_mqtt.h>
+#include <xively.h>
+#include <xi_types.h>
+
```

Comment:
what's the difference between the gateway layer and the gateway glue layer?

Reply:
see above comment, they are at different ends of the edge device layer chain, or gateway layer chain, maybe I'll rename it.

Reply:
yep. 👍